ops/services/app_functions/report_stream_batched_publisher/functions/QueueBatchedReportStreamUploader/lib.ts

```diff
@@ -206,6 +206,10 @@ export async function reportExceptions(
   );
 }
 
+export async function publishToFileErrorQueue(queueClient: QueueClient,  messages: DequeuedMessageItem[]) {
```

Comment:
This function is not named correctly, you can potentially use this function to send messages to any queue, since it takes the queue client as a param, better name might be `publishToQueue`